doc/api/process.markdown

```diff
@@ -159,40 +159,60 @@ event:
       return reportToUser(JSON.pasre(res)); // note the typo
     }); // no `.catch` or `.then`
 
+Here is an example of a coding pattern that will also trigger
+`'unhandledRejection'`:
+
+    function SomeResource() {
+      // Initially set the loaded status to a rejected promise
+      this.loaded = Promise.reject(new Error('Resource not yet loaded!'));
```

Comment:
I have not shown a complete implementation of SomeResource. I could do so if it would help. Essentially there's a SomeResource.prototype.load() method that will replace the promise with a successful one.

The idea is that anyone doing `.loaded.then(...)` before anything is loaded goes down the error path. Whereas, after something is loaded, they go down the success path.